src/schema/metadata/EpochLength.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: EpochLength
+description: |
+  Duration of individual epochs in seconds (for example, 1)
+  in case of epoched data.
+  If recording was continuous or discontinuous, leave out the field.
```

Comment:
This sentence is only in the iEEG table. I'm not sure if it's iEEG-specific or not.

Reply:
This sentence could also go to EEG / MEG --> the field SHOULD NOT be specified if the recording is not epoched (i.e., if it is continuous or discontinuous).

So rather than delete it from iEEG for consistency, add it to EEG and MEG for consistency

Reply:
That's good to hear. The definition above is applied to all instances of the field, so in the current setup it's applied to EEG and MEG automatically.